AppController/djinn.rb

```diff
@@ -5515,39 +5499,60 @@ def try_to_scale_up(app_name)
           Djinn.log_debug("#{host} CPUs are too busy.")
           break
         end
-        available_hosts << host
+
+        # We add the host as many times as AppServers it can run.
+        (max_new_tot > max_new_free ? max_new_free : max_new_tot).downto(1) {
```

Comment:
Would you mind renaming the max_new_tot variable to max_new_total, I usually prefer using the full word if its not too long to begin with.